test/ObserverSpec.ts

```diff
@@ -1,8 +1,12 @@
 import {expect} from "chai";
 import {SampleObserver} from "./fixture/SampleObserver";
 import {ServiceStatus} from "../src/ServiceStatus";
-describe("@Observer", () => {
-    it("should update state of observer when state of subject changes", () => {
+import {SampleObserverOne} from "./fixture/SamleObserverOne";
+
+describe("@ObserverStatusForOneClass", () => {
+    afterEach(() => ServiceStatus.instance.cancelInterval());
```

Comment:
this can be a before block, since we need to cancel only once before.
also instead of calling instance.cancelInterval(), we can make cancelInterval public and static, and make the isntance private.